packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -139,6 +139,37 @@ export default {
             commit('UPDATE_AVAILABLE_FULFILMENT_TIMES', data);
         },
 
+        /**
+         * Get the basket details from the backend and update the state.
+         *
+         * @param {Object} commit - Automatically handled by Vuex to be able to commit mutations.
+         * @param {Object} payload - Parameter with the different configurations for the request.
+         */
+        getBasket: async ({ commit }, {
+            url,
+            tenant,
+            language,
+            timeout
+        }) => {
+            // TODO: deal with exceptions.
+            const config = {
+                method: 'get',
+                headers: {
+                    'Content-Type': 'application/json',
+                    'Accept-Tenant': tenant,
+                    'Accept-Language': language
+                },
+                timeout
+            };
+
+            const { data } = await axios.get(url, config);
+            const basketDetails = {
+                serviceType: data.ServiceType.toLowerCase()
```

Comment:
Do you need to check that data exists before trying to access it? Will it always be available? If not you might get an error at `ServiceType`/

Reply:
It should always be available